File: 1.js
let S_P = 1872,
  S_N = 2167,
  S_TP = 383;

let M_P = 1076,
  M_N = 1381,
  M_TP = 522;

let p = (S_TP + M_TP) / (S_P + M_P);
let r = (S_TP + M_TP) / (S_N + M_N);
let f1 = (2 * p * r) / (p + r);

console.log(p, r, f1);

File: 1.py
M_P, M_N, M_TP = 1076, 1381, 625
S_P, S_N, S_TP = 1089, 1699, 233

p = (M_TP+S_TP)/(M_P+S_P)
r = (M_TP+S_TP)/(M_N+S_N)
f1 = (2.0 * p * r) / (p + r)
print(p, r, f1)